api/libproject/devstart.rb

```diff
@@ -298,10 +298,11 @@ def drop_cloud_cdr(*args)
   end
 end
 
-def run_cdr_data_migrations(*args)
+def run_local_data_migrations(*args)
```

Comment:
Just to confirm: we don't think we need to run these data migrations for cloud?

Reply:
It probably makes sense to be able to run this insert of the cdr_version in the cloud. I will add it once i sort out problems that introduced in peter's PR.